scripts/test_ports/vcpkg-ci-aurora-au/project/test_au_client.cc

```diff
@@ -0,0 +1,17 @@
+#include <iostream>
+
+#include "au/au.hh"
+#include "au/constants/speed_of_light.hh"
+#include "au/io.hh"
+#include "au/units/meters.hh"
+#include "au/units/seconds.hh"
+
+using ::au::SPEED_OF_LIGHT;
+using ::au::symbols::m;
+using ::au::symbols::s;
+
+int main(int argc, char **argv) {
+  std::cout << "Speed of light in m/s: " << SPEED_OF_LIGHT.as<int>(m / s)
+            << std::endl;
+  return 0;
+}

```

Comment:
I just remembered that we can make suggestions as comments, which you can then commit directly!  Let's see if this will fix the remote builds (it's the changes from https://github.com/chiphogg/vcpkg/commit/894bb2355040b01cd463f18a73816c7157c0df45).

```suggestion
#include "au/io.hh"
#include "au/units/hours.hh"
#include "au/units/meters.hh"
#include "au/units/miles.hh"

using ::au::symbols::h;
using ::au::symbols::mi;
constexpr auto km = ::au::kilo(::au::symbols::m);

int main(int argc, char **argv) {
  constexpr auto v = 65.0 * mi / h;
  std::cout << v << ", in km/h, rounded to nearest integer, is "
            << round_as(km / h, v) << std::endl;
  return 0;
}
```